README.org

```diff
@@ -69,6 +65,24 @@ There is an option to have your user credentials (email address and password) sa
 The default is not to do this because if not properly configured it would save these unencrypted which is not a good default to have.
 Customize the variable =mastodon-auth-source-file= if you want to enable this feature.
 
+*** 2 Factor Auth
+**** Overview
+After setting up =mastodon-instance-url= set the following in your
+=.emacs= or =customize=:
+
+1. =mastodon-auth-mechanism=
+
+**** Set up
+#+BEGIN_SRC emacs-lisp
+  (setq mastodon-auth-mechanism "oauth2")
```

Comment:
Actually it seem that  `mastodon-auth-mechanism` should be a symbol (that's what you define as options in the defcustom.  So this should be `'oauth2` and not `"oauth2"`.

It would be clearer, if you had consistently use `(eq 'oauth2 mastodon-auth-mechanism)` instead of doing a `(string= ...`.